src/modules/navigation/partials/NavigationTableItem.vue

```diff
@@ -193,6 +219,18 @@ export default {
 				})
 			}
 		},
+		async toggleArchiveTable(archived) {
+			const res = await this.$store.dispatch('updateTable', {
+				id: this.table.id,
+				data: { archived },
+			})
+
+			// eslint-disable-next-line no-console
+			if (!res) { console.log('failed to archive/unarchive table') }
+
+			// eslint-disable-next-line no-console
+			console.log('archived/unarchived table')
```

Comment:
Can probably dropped, the vuex store method should already log a console error if this fails.